docker/deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
Cool. I just learned about this from a little article here:

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/